test/parallel/test-fs-glob.mjs

```diff
@@ -338,6 +338,9 @@ describe('fsPromises glob', function() {
   }
 });
 
+const normalizeDirent = (dirent) => relative(fixtureDir, join(dirent.parentPath, dirent.name));
+const normalizePath = (path) => (isAbsolute(path) ? relative(fixtureDir, path) : join(path));
```

Comment:
```suggestion
const normalizePath = (path) => (isAbsolute(path) ? relative(fixtureDir, path) : path);
```

Reply:
The `join` is important. It verifies that the path seperators are applied correctly.

```js
path.win32.join("hello/world") // "hello\\world"
```

Reply:
Can you add a comment?